### DISCUSSION TITLE
Dashboard Widget

### DISCUSSION THREAD

**Discussion by plozmun** [sulu/sulu#5708] [Ideas]
Sulu backend is clean and easy to use, but maybe dashboard can be improved with some widgets to help content managers to accelerate their workflow.

Some examples:

- Drafts 
- Last published or update pages ( Content editor can access quickly to its last created pages ) 
- Custom Widgets ( Last Events, Form Data , etc ..)

**Comment by danrot** [comment]
@plozmun That's a great idea, that has already been brought up a while ago in [this issue](https://github.com/sulu/sulu/issues/2815). Would be nice if you can add your ideas there :slightly_smiling_face:

**Reply by plozmun** [reply]
Sure!
